scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -151,25 +151,45 @@ <h3 class="modal__heading" ng-show="desk.edit._id" translate>Edit "{{desk.edit.n
                         </div>
                     </div>
                     <div class="form__row">
-                        <div class="sd-line-input sd-line-input--is-select">
+                        <div class="sd-line-input sd-line-input--required">
                             <label class="sd-line-input__label" translate>Default Content Template</label>
-                            <select
-                                ng-model="desk.edit.default_content_template"
-                                ng-options="template._id as template.template_name for template in templates"
-                                class="sd-line-input__select"
-                                data-test-id="field--default-content-template"
-                            >
+                            <div class="sd-alert sd-alert--hollow sd-alert--small">
+                                <i class="icon-info-sign icon--blue icon--2x sd-alert__icon"></i>
+                               <span>
+                                   {{ :: 'Select a default template for new articles. If you don\'t see any, first make sure that there is at least one content profile. Each time a profile is created, Superdesk creates a template with the same name. ' | translate }}
```

Comment:
You can use `<div translate>x</div>` instead of `<div>{{:: 'x' | translate}}<div>`. It's shorter and doesn't require escaping quotes. The filter is mostly used for translating attributes.